### PR TITLE
Fix of undefined error message

### PR DESCRIPTION
**Pull request by Langriklol**
- bug fix / new feature?   huge feature
- BC break? of course not

### CODE REVIEW

File: template/@layout.phtml [hunk 1]
```diff
@@ -114,7 +114,14 @@ foreach ($staticAssets as $staticAsset) {
...
         return Promise.resolve(ok);
     }, (error) => {
-        let data = error.response.data;
+        if (typeof error === 'object' || error.response === undefined) {
+	    app.$bvToast.toast(error.toString(), {
+		    variant: 'danger',
+		    title: 'Error',
+		    toaster: 'b-toaster-bottom-left'
+	    });
+	return;
```
Comment (janbarasek) [inline template/@layout.phtml:123]: pls spaces

File: template/@layout.phtml [hunk 2]
```diff
@@ -114,7 +114,14 @@ foreach ($staticAssets as $staticAsset) {
         }
         return Promise.resolve(ok);
     }, (error) => {
-        let data = error.response.data;
```
Comment (janbarasek) [inline template/@layout.phtml:117]: please revert this line

**Review by janbarasek — CHANGES_REQUESTED** [review]

**Review by Langriklol — COMMENTED** [review]
CS fixed

File: template/@layout.phtml [hunk 3]
```diff
@@ -114,7 +114,15 @@ foreach ($staticAssets as $staticAsset) {
...
+        if (typeof error === 'object' || error.response === undefined) {
+            app.$bvToast.toast(error.toString(), {
+                    variant: 'danger',
+                    title: 'Error',
+                    toaster: 'b-toaster-bottom-left'
```
Comment (janbarasek) [inline template/@layout.phtml:121]:
```suggestion
                variant: 'danger',
                title: 'Error',
                toaster: 'b-toaster-bottom-left'
```

File: template/@layout.phtml [hunk 4]
```diff
@@ -114,7 +114,15 @@ foreach ($staticAssets as $staticAsset) {
         }
         return Promise.resolve(ok);
     }, (error) => {
-        let data = error.response.data;
+        if (typeof error === 'object' || error.response === undefined) {
```
Comment (janbarasek) [inline template/@layout.phtml:117]:
```suggestion
        if (typeof error === 'object' && error.response === undefined) {
```

Comment (janbarasek) [inline template/@layout.phtml:117]:
```suggestion
        if (typeof error === 'object' && error.response === undefined) {
```

**Comment by janbarasek** [comment]
Nice to have: https://stackoverflow.com/questions/1340872/how-to-get-javascript-caller-function-line-number-and-caller-source-url

File: template/@layout.phtml [hunk 5]
```diff
@@ -114,7 +114,15 @@ foreach ($staticAssets as $staticAsset) {
...
-        let data = error.response.data;
+        if (typeof error === 'object' && error.response === undefined) {
+            app.$bvToast.toast(error.toString(), {
+                variant: 'danger',
+                title: 'Error',
+                toaster: 'b-toaster-bottom-left'
+	    });
+            return;
+        }
+        let data = error.response.data
```
Comment (janbarasek) [inline template/@layout.phtml:125]:
```suggestion
        let data = error.response.data;
```

**Pull request merged**